definitions/npm/react-helmet_v6.x.x/flow_v0.104.x-/react-helmet_v6.x.x.js

```diff
@@ -36,8 +36,11 @@ declare module 'react-helmet' {
     static canUseDOM: boolean;
   }
   declare var HelmetExport: typeof Helmet;
-  declare export { HelmetExport as Helmet };
-  declare export default typeof HelmetExport;
+  declare module.exports: {
+    Helmet: typeof HelmetExport,
+    canUseDOM: boolean,
+    default: typeof HelmetExport,
+  };
```

Comment:
I think what you had before was pretty close. But I had a quick look at docs plus what's in TS specs and to import `Helmet` you do 
```
import {Helmet} from 'react-helmet';
// or
import Helmet from 'react-helmet';
```

So what you're more looking for is.

```suggestion
  declare export var Helmet: typeof HelmetExport;
  declare export var canUseDOM: boolean;
  declare module.exports: typeof HelmetExport;
```

If you have it as an object it basically means if you do 
```
import Helmet from 'react-helmet';

Helmet.default // you're importing an object that has the properties but probably doesn't reflect what you want
```

Reply:
Totally went for `describe` and in v3 and v5 as well